statistics/ExecutionSummaryCliParser.go

```diff
@@ -0,0 +1,106 @@
+package statistics
+
+import (
+	"regexp"
+	"strconv"
+)
+
+//ExecutionSummaryCliParser ...
+type ExecutionSummaryCliParser struct {
+}
+
+//Parse ...
+func (instance ExecutionSummaryCliParser) Parse(data string) ExecutionSummary {
+	return ExecutionSummary{
+		RunningTime:   parseRunningTime(data),
+		Throughput:    parseThroughput(data),
+		TotalRequests: parseTotalRequests(data),
+		TotalErrors:   parseTotalErrors(data),
+		Availability:  parseAvailability(data),
+		Bytes: ByteSummary{
+			TotalSent:     parseTotalBytesSent(data),
+			TotalReceived: parseTotalBytesReceived(data),
+		},
+		MeanResponseTime: parseMeanResponseTime(data),
+		MinResponseTime:  parseMinResponseTime(data),
+		MaxResponseTime:  parseMaxResponseTime(data),
+	}
+}
+
+func checkMatchString(match []string) string {
+	if len(match) != 2 {
+		return ""
+	}
+	return match[1]
+}
+
+func checkMatchFloat(match []string) float64 {
+	if len(match) != 2 {
+		return float64(-1)
+	}
+	value, _ := strconv.ParseFloat(match[1], 64)
+	return value
+}
+
+func checkMatchInt64(match []string) int64 {
+	if len(match) != 2 {
+		return int64(-1)
+	}
+	value, _ := strconv.Atoi(match[1])
+	return int64(value)
+}
+
+func parseRunningTime(data string) string {
+	var runningTime = regexp.MustCompile(`Running Time: ([\d\w\.]+)`)
```

Comment:
You could refactor this slightly to move the `regexp` `MustCompile` and `FindStringSubmatch` to inside the checkMatchXXX methods resulting in:

```go
return checkMatchString(`Running Time: ([\d\w\.]+)`, data)
```